setup.py

```diff
@@ -19,22 +19,22 @@
     download_url='http://github.com/pzinemanas/DCASE-models/releases',
     packages=setuptools.find_packages(),
     install_requires=[
-        'numpy==1.17.4',
-        'pandas==0.25.3',
-        'SoundFile==0.10.3.post1',
-        'PyYAML==5.2',
-        'numba==0.48.0',
-        'librosa==0.7.1',
-        'openl3==0.3.1',
-        'scikit-learn==0.22.1',
-        'keras==2.2.4',
-       # 'tensorflow-gpu==1.12.0',
-        'autopool==0.1.0',
-        'wget==3.2',
-        'sox==1.3.7',
-        'sed_eval==0.2.1',
+        'numpy>=1.1',
+        'pandas>=0.25',
```

Comment:
Do we still need pandas?

Reply:
No, we have to implement a `read_csv` function which is the only thing you are using from pandas.